index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio Project</title>
```

Comment:
- [ ] Kindly change your page title to something that describe the content of the page very well. Something like `Kajal Portfolio` or `Kajal full-stack webdeveloper` is more appropriate than `Portfolio Project`

Reply:
@Qoosim I think the review is not valid. All of the code reviewers ignore these color contrast you told me to change. Everybody using the figma template colors, which also fails the color contrast test.
here is an example also, please you can check...
https://github.com/Willian-Berrocal/Portfolio-project/pull/9

I will appreciate it if better light is shown on how to implement the color contrast correction. Thanks.

Reply:
Hi @Kajal19-del, color contrast check is one of the requirements we test for while reviewing this milestone of your project. The fact that some code reviewers ignore this does not render this review invalid as you've qualified it. I have reviewed a lot of projects on this milestone which the owners manage to make their color contrast pass. You can request for another review. If the next reviewer ignore it or not, all the best. But running the color contrast check on your page is what we're asked to do as part of review. Thanks